### ISSUE TITLE
Can I train Lora at automatic111 SD runing on Paperspace? (or other external GPU)?

### ISSUE THREAD

**Issue by KttKrl** [bmaltais/kohya_ss#263]
If I run made fune tuning extention in SD on paperspace I got whis error 

A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'
usage: fine_tune.py [-h] [--v2] [--v_parameterization]
[--pretrained_model_name_or_path PRETRAINED_MODEL_NAME_OR_PATH]
[--train_data_dir TRAIN_DATA_DIR] [--shuffle_caption]
[--caption_extension CAPTION_EXTENSION]
[--caption_extention CAPTION_EXTENTION]
[--keep_tokens KEEP_TOKENS] [--color_aug] [--flip_aug]
[--face_crop_aug_range FACE_CROP_AUG_RANGE]
[--random_crop] [--debug_dataset]
[--resolution RESOLUTION] [--cache_latents]
[--enable_bucket] [--min_bucket_reso MIN_BUCKET_RESO]
[--max_bucket_reso MAX_BUCKET_RESO]
[--bucket_reso_steps BUCKET_RESO_STEPS]
[--bucket_no_upscale]
[--caption_dropout_rate CAPTION_DROPOUT_RATE]
[--caption_dropout_every_n_epochs CAPTION_DROPOUT_EVERY_N_EPOCHS]
[--caption_tag_dropout_rate CAPTION_TAG_DROPOUT_RATE]
[--in_json IN_JSON] [--dataset_repeats DATASET_REPEATS]
[--output_dir OUTPUT_DIR] [--output_name OUTPUT_NAME]
[--save_precision {None,float,fp16,bf16}]
[--save_every_n_epochs SAVE_EVERY_N_EPOCHS]
[--save_n_epoch_ratio SAVE_N_EPOCH_RATIO]
[--save_last_n_epochs SAVE_LAST_N_EPOCHS]
[--save_last_n_epochs_state SAVE_LAST_N_EPOCHS_STATE]
[--save_state] [--resume RESUME]
[--train_batch_size TRAIN_BATCH_SIZE]
[--max_token_length {None,150,225}] [--mem_eff_attn]
[--xformers] [--vae VAE]
[--max_train_steps MAX_TRAIN_STEPS]
[--max_train_epochs MAX_TRAIN_EPOCHS]
[--max_data_loader_n_workers MAX_DATA_LOADER_N_WORKERS]
[--persistent_data_loader_workers] [--seed SEED]
[--gradient_checkpointing]
[--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS]
[--mixed_precision {no,fp16,bf16}] [--full_fp16]
[--clip_skip CLIP_SKIP] [--logging_dir LOGGING_DIR]
[--log_prefix LOG_PREFIX] [--noise_offset NOISE_OFFSET]
[--lowram]
[--save_model_as {None,ckpt,safetensors,diffusers,diffusers_safetensors}]
[--use_safetensors] [--optimizer_type OPTIMIZER_TYPE]
[--use_8bit_adam] [--use_lion_optimizer]
[--learning_rate LEARNING_RATE]
[--max_grad_norm MAX_GRAD_NORM]
[--optimizer_args [OPTIMIZER_ARGS ...]]
[--lr_scheduler LR_SCHEDULER]
[--lr_warmup_steps LR_WARMUP_STEPS]
[--lr_scheduler_num_cycles LR_SCHEDULER_NUM_CYCLES]
[--lr_scheduler_power LR_SCHEDULER_POWER]
[--diffusers_xformers] [--train_text_encoder]
fine_tune.py: error: unrecognized arguments: --prior_loss_weight 1.0
